src/Model/CronModel.php

```diff
@@ -62,6 +62,25 @@ public function findMostRecentRuns (WrappedJob $job, int $limit) : array
     }
 
 
+    public function removeOldLogsByStorageDuration (int $storageDuration) : bool
+    {
+        $compareDate = new \DateTimeImmutable("- {$storageDuration} days");
+
+        try {
+            $this->entityManager->createQueryBuilder()
+                ->delete(CronJobRun::class, "job")
+                ->where("job.timeRun < :storageDuration")
+                ->setParameter("storageDuration", $compareDate)
+                ->getQuery()
+                ->execute();
+        } catch (\Exception $e) {
+            return false;
+        }
+
+        return true;
```

Comment:
Please move the try-catch block to the calling side of this method. And rename `$storageDuration` to `$logTtl`.